events/api.py

```diff
@@ -3634,38 +3626,46 @@ def get_serializer_context(self):
             {"headline", "secondary_headline"}
         )
         context["extensions"] = get_extensions_from_request(self.request)
-        (
-            context["keywords"],
-            context["location"],
-            context["image"],
-            context["sub_events"],
-            bulk,
-        ) = self.cache_related_fields(self.request)
+
+        if self.request.method in ("POST", "PUT", "PATCH"):
```

Comment:
I wonder if `not in permissions.SAFE_METHODS` would be another option for this. That of course would include more than just the mentioned methods, so maybe this is better. :+1: